service/storage_service.cc

```diff
@@ -2413,12 +2441,12 @@ future<> storage_service::removenode(locator::host_id host_id, std::list<locator
                 }).get();
 
 
-                // Step 5: Announce the node has left
+                // Step 6: Announce the node has left
```

Comment:
I.e., call ```ss._group0->make_nonvoter(raft_id).get()``` right before this step.

Reply:
See my answer above, I want `make_nonvoter` to happen early in `removenode`.